src/dev/riscv/LupV.py

```diff
@@ -45,24 +46,20 @@ class LupV(Platform):
     """LupV Platform copied from dev/riscv/HiFive.py
     This platform will continually be upadated to
     incorporate the LupIO devices.
-
```

Comment:
Nit: Probably don't need to delete these lines, but it's up to you.

Reply:
In any case, it should probably be a separate commit since it's unrelated to adding the block device to this platform.